package.json

```diff
@@ -63,11 +63,12 @@
     "grid3_client": "1.3.1",
     "js-yaml": "^4.1.0",
     "marked": "^3.0.8",
+    "node-rsa": "^1.1.1",
```

Comment:
this package is not used anywhere. if not needed can we remove it and revert changes in `yarn.lock` file